mne_bids/utils.py

```diff
@@ -264,6 +264,30 @@ def make_dataset_description(path, name=None, data_license=None,
     _write_json(description, fname, verbose=verbose)
 
 
+def age_on_date(bday, exp_date):
+    """
+    Determine the age of someone from their birthday and the date of the
+    experiment.
```

Comment:
Make this a one line description and fit it after the quotation marks. Cool function by the way :)